aws/data_source_aws_iam_group.go

```diff
@@ -58,6 +82,20 @@ func dataSourceAwsIAMGroupRead(d *schema.ResourceData, meta interface{}) error {
 	d.Set("arn", group.Arn)
 	d.Set("path", group.Path)
 	d.Set("group_id", group.GroupId)
+	d.Set("members", dataSourceUsersRead(resp.Users))
```

Comment:
When using `d.Set()` with aggregate types (TypeList, TypeSet, TypeMap), [we should perform error checking to prevent issues where the code is not properly able to set the Terraform state](https://www.terraform.io/docs/extend/best-practices/detecting-drift.html#error-checking-aggregate-types). 👍 

```suggestion
	if err := d.Set("members", dataSourceUsersRead(resp.Users)); err != nil {
		return fmt.Errorf("error setting members: %s", err)
	}
```

Reply:
fixed